src/ui/map.js

```diff
@@ -2618,16 +2622,29 @@ class Map extends Camera {
         return this.style.getFeatureState(feature);
     }
 
-    _containerDimensions() {
-        let width = 0;
-        let height = 0;
-
+    _containerDimensions(): any {
```

Comment:
maybe rename this to `_updateContainerDimensions()` so thats its clearer that this function is intended as a thing that updates values in the `this` scope?